netty/src/main/java/io/grpc/netty/InternalProtocolNegotiators.java

```diff
@@ -31,6 +33,36 @@ public final class InternalProtocolNegotiators {
 
   private InternalProtocolNegotiators() {}
 
+  /**
+   * Returns a {@link ProtocolNegotiator} that ensures the pipeline is set up so that TLS will
+   * be negotiated, the {@code handler} is added and writes to the {@link io.netty.channel.Channel}
+   * may happen immediately, even before the TLS Handshake is complete.
+   * @param executorPool a dedicated {@link Executor} pool for time-consuming TLS tasks
```

Comment:
You don't utilize or store `executorPool`

Reply:
For some reason, it didn't get included in original commit, fixed now.